### PR TITLE
added a frequently asked question about Lerna

### PR DESCRIPTION
**Pull request by mquellhorst**
A question I got a couple of times already came up again so I've added it to the docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ We're using the following stack:
 
 - [Jest](https://jestjs.io/)
 - [Prettier](https://prettier.io/)
-- [Lerna](https://lerna.js.org/)
+- [Lerna](https://github.com/lerna/lerna/tree/main/commands/add#readme)
```
Comment (alalonde) [inline README.md:26]: I think the original link is more appropriate here

File: README.md [hunk 2]
```diff
@@ -66,6 +66,12 @@ Initial TS Compilation For Monorepo Packages
 yarn typecheck
 ```
 
+Adding a package to the codebase can be done with Lerna
```
Comment (alalonde) [inline README.md:69]: That new link should probably go here instead

**Review by alalonde — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mquellhorst) [reply README.md:26]: I noticed the website itself doesn't contain much info for commands as `lerna add ...`
But yeah if I put the link at the other spot in the documentation sounds good. 
Thanks!

File: README.md (hunk 2 above)
Comment (mquellhorst) [reply README.md:69]: Thanks for the feedback!

**Comment by alalonde** [comment]
Waiting on @mquellhorst for requested changes..

**Comment by mquellhorst** [comment]
> 
> 
> Waiting on @mquellhorst for requested changes..

Pushed the changes and added a some extra information about the --no-verify option

File: README.md [hunk 3]
```diff
@@ -66,6 +66,12 @@ Initial TS Compilation For Monorepo Packages
...
+
+```shell script
+yarn lerna add <package> packages/<folder> [--dev] [--exact] [--peer]
+```
```
Comment (alalonde) [inline README.md]: This inline example seems like it would be useful

**Review by alalonde — APPROVED** [review]

**Pull request merged**